.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo" 
+    directory: "/" 
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning-background-processor" 
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning-block-sync" 
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning-invoice" 
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning-net-tokio"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning-persister"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "cargo" 
+    directory: "/lightning"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions" 
+    directory: "/lightning"
```

Comment:
Our CI is in the root of the repo (in `.github`), so shouldn't this be "/"?

Reply:
Yes, that is the default location that GitHub looks for actions , here is an example https://github.com/ossf/scorecard/blob/4837262895fa4ebc4438c94038facfee4529e4bf/.github/dependabot.yml#L10

Reply:
Right, but the directory here says "/lightning"? So wouldn't this look int he wrong place, then?

Reply:
:face_palm

Reply:
I will fix this.

Reply:
Fixed it. Thanks